cartoframes/data/clients/sql_client.py

```diff
@@ -163,30 +200,29 @@ def create_table(self, table_name, columns, cartodbfy=True):
             COMMIT;
         '''.format(
             drop='DROP TABLE IF EXISTS {}'.format(table_name),
-            create='CREATE TABLE {0} ({1})'.format(table_name, columns),
+            create='CREATE TABLE {0} ({1})'.format(table_name, ','.join(columns)),
             cartodbfy='SELECT CDB_CartoDBFyTable(\'{0}\', \'{1}\')'.format(
                 schema, table_name) if cartodbfy else ''
         )
-        return self.execute(query)
+        self.execute(query)
 
     def insert_table(self, table_name, column_names, column_values):
         """Insert a row to the table.
 
         Args:
             table_name (str): name of the table.
-            column_names (str, list of str): names of the columns.
-            column_values (str, list of str): values of the columns.
+            column_names (list of str): names of the columns.
+            column_values (list of tuple): values of the columns.
+
+        Example:
+            >>> sql.insert_table('table_name', ['column1', 'column2'], [('value1', 1), ('value2', 2)])
 
         """
-        if isinstance(column_names, str):
-            column_names = [column_names]
-        if isinstance(column_values, str):
-            column_names = [column_values]
-        sql_values = [self._sql_format(x) for x in column_values]
+        sql_values = [self._row_values_format(row_values) for row_values in column_values]
         query = '''
-            INSERT INTO {0} ({1}) VALUES({2});
+            INSERT INTO {0} ({1}) VALUES {2};
         '''.format(table_name, ','.join(column_names), ','.join(sql_values))
-        return self.execute(query)
+        self.execute(query)
 
     def update_table(self, table_name, column_name, column_value, condition):
```

Comment:
Great! it'd be awesome to automatically parse the condition but the example makes it easy. LGTM